components/api/db/cellery-hub-db.bal

```diff
@@ -34,10 +34,25 @@ public function getOrganization (string orgName) returns json | error {
         return resPayload;       
     } else {
         log:printDebug(io:sprintf("The requested organization \'%s\' was not found in REGISTRY_ORGANIZATION", orgName));
+        res.close();
         return null;
     }
 }
 
+public function getOrganizationAvailability(string orgName) returns boolean | error {
```

Comment:
Since we are anyway using the same query, why can't we use getOrganization() method itself (without implementing this method) and see if null or an actual org is returned ?

Reply:
It may costly since getOrganization() method is converting the resulting table into OrgResponse. Hence used a method only checking the orgName availability.